jax/_src/interpreters/pxla.py

```diff
@@ -2874,10 +2876,12 @@ def lower_sharding_computation(
     out_shardings: Union[Sequence[Union[sharding_internal.XLACompatibleSharding, UnspecifiedValue]], UnspecifiedValue],
     donated_invars: Sequence[bool],
     global_in_avals: Sequence[core.ShapedArray],
+    *,
```

Comment:
The star is simply to force this to be a kwarg, and to make it a mistake to pass is as positional. It seems that many of the arguments we pass as keyword and forcing that is safer, because you do not allow Python to mistakenly interpret the as positional. wdyt?

Reply:
Leaving in this CL per offline discussion